src/creationView/CreateView.js

```diff
@@ -0,0 +1,2316 @@
+import { Localizer, ActionHelper } from "../common/ActionSdkHelper";
+
+let request;
+let questionCount = 0;
```

Comment:
Unused questionCount please remove, also please took care of other unused code & cleanUp